app/src/main/java/ke/co/svs/mykrypto/domain/database/RoomDatabase.kt

```diff
@@ -32,14 +30,4 @@ fun List<CryptoDetailsResponse>.asCryptoDetailsModel(): List<CryptoDetails> {
             lastUpdated = it.lastUpdated.toString()
         )
     }
-}
-
-fun List<CryptoResponse>.asCryptoModel(): List<Crypto> {
-    return map {
```

Comment:
Not sure why this method was removed. Is the intention not to use crypto details?